docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/Ubuntu/README.md

```diff
@@ -86,6 +86,22 @@ make      // For production environments
 make dev  // For development environments
 ```
 
+## Creating the lorisadmin user
+```bash
+# Create lorisadmin user and group
+# Give lorisadmin `sudo` permission. This is required for the install process
+# in order to automatically generate Apache configuration files.
+# Sudo privileges can be revoked once the install is completed.
+sudo useradd -U -m -G sudo -s /bin/bash lorisadmin
+# Add apache to the lorisadmin group
+sudo usermod -a -G lorisadmin www-data
+# Set the password for the lorisadmin account
+sudo passwd lorisadmin
+sudo mkdir -m 755 -p /var/www/$projectname
```

Comment:
let's make the root directory consistent across this readme. i.e. `/var/www/loris`